key/keyutil/keyutil.go

```diff
@@ -0,0 +1,51 @@
+package keyutil
```

Comment:
suggestion: could move to the `key` package into a `rand.go` file.

If it's just used for testing we could also move it to the `testutil` package.

Reply:
I think I will move to testutil. I don't think we want to encourage creating random keys with math/rand for anything other than tests